tests/test_gbdesAstrometricFit.py

```diff
@@ -1136,6 +1140,116 @@ def test_useColor(self):
+                "pmDec": self.pmDecs,
+                "parallax": parallaxes,
+                "MJD": self.mjds,
+            }
+        )
+        raCorrection, decCorrection = calculate_apparent_motion(data, self.refEpoch)
+
+        pmRACosDec = self.pmRAs * np.cos(self.decs)
+        coords = SkyCoord(
+            ra=self.ras,
+            dec=self.decs,
+            pm_ra_cosdec=pmRACosDec,
+            pm_dec=self.pmDecs,
+            obstime=self.refEpoch,
+        )
+
+        newCoords = coords.apply_space_motion(new_obstime=self.mjds)
+        astropyDRA = newCoords.ra.degree - coords.ra.degree
+        astropyDDec = newCoords.dec.degree - coords.dec.degree
+
+        self.assertFloatsAlmostEqual(astropyDRA, raCorrection.value, rtol=1e-6)
+        self.assertFloatsAlmostEqual(astropyDDec, decCorrection.value, rtol=1e-6)
+
+    def test_parallax(self):
+        """Calculate the change in position due to parallax for a given time
+        change and compare the results against astropy. Astropy will not give
+        the parallax part separate from other sources of space motion, such as
+        annual aberration, but it can be obtained by comparing the calculated
+        positions of an object with a given parallax and one with a much
+        further distance. The result is still only approximately the same, but
+        is close enough for accuracy needed here.
+        """
+        # Set proper motions to zero to test parallax correction by itself.
+        pms = np.zeros(len(self.ras)) * u.mas / u.yr
+
+        data = Table(
+            {
+                "ra": self.ras,
+                "dec": self.decs,
+                "pmRA": pms,
+                "pmDec": pms,
+                "parallax": self.parallaxes,
+                "MJD": self.mjds,
+            }
+        )
+        raCorrection, decCorrection = calculate_apparent_motion(data, self.refEpoch)
+
+        coords = SkyCoord(
+            ra=self.ras,
+            dec=self.decs,
+            pm_ra_cosdec=pms,
+            pm_dec=pms,
+            distance=Distance(parallax=self.parallaxes),
+            obstime=self.refEpoch,
+        )
+        # Make another set of objects at the same coordinates but a much
+        # greater distant in order to isolate parallax when applying space
+        # motion.
+        refCoords = SkyCoord(
+            ra=self.ras,
+            dec=self.decs,
+            pm_ra_cosdec=pms,
+            pm_dec=pms,
+            distance=1e10 * Distance(parallax=self.parallaxes),
+            obstime=self.refEpoch,
+        )
+
+        newCoords = coords.apply_space_motion(new_obstime=self.mjds).transform_to("gcrs")
+        refNewCoords = refCoords.apply_space_motion(new_obstime=self.mjds).transform_to("gcrs")
+        astropyDRA = newCoords.ra.degree - refNewCoords.ra.degree
+        astropyDDec = newCoords.dec.degree - refNewCoords.dec.degree
+
+        self.assertFloatsAlmostEqual(astropyDRA, raCorrection.value, rtol=1e-1)
```

Comment:
As I noted above, you should do the comparison in `mas` and probably with `atol`, so we can see how accurate it is in "real" units. It's not obvious to me what a 10% accuracy means here, but it doesn't feel like it's very good?

Reply:
Well, the actual accuracy is about 1.1% at worst, but I rounded up rather than have the value be too tuned. I guess I could do 2e-2. For the atol, I'm not sure that it's preferable here, since you don't know what the values of `astropyDRA` and `raCorrection` are here. I'm going to just add a comment on the scale of the errors here, which correspond to 0.04 mas at most.

Reply:
Ah, thank you for that clarification. That's not as bad as I'd thought.